README.md

```diff
@@ -48,6 +48,12 @@ Cores 0, 1, 2, 3 are E-cores, the others are P-cores.
 
 ![Apple M4 Max (16C) Core-to-Core Latency](results/241208-1-m4max-steady_clock-i-2000-r-300.png?raw=true "Apple M4 Max (16C) Core-to-Core Latency")
 
+## Apple M3 Max (16C)
+
+Cores 0, 1, 2, 3 are E-cores, the others are P-cores.
+
+![Apple M2 Pro (10C) Core-to-Core Latency](results/241212-0-m3max.png?raw=true "Apple M3 Max (16C) Core-to-Core Latency")
```

Comment:
typo M2 Pro (10C)

Reply:
I've followed the instructions in README and I have just one results.log, should I rename it before commit ?

Reply:
No that's all good. It's just the text (alt text?) above is a copy&paste from m2 pro. Just a tiny edit.
I couldn't figure out how to checkout your branch and fix it myself. Maybe I can't if it comes from a fork.

Reply:
Yes rename the results.log the same as the png please

Reply:
Sorry for the typo, I've added results log